public/app/features/profile/UserProfileEdit.tsx

```diff
@@ -27,7 +27,7 @@ export const UserProfileEdit: FC<Props> = ({ navModel }) => (
         teams: Team[],
         orgs: UserOrg[],
         sessions: UserSession[],
-        user: UserDTO
+        user?: UserDTO
```

Comment:
Why do we need to make this optional and then assert it's not later?

Reply:
Because it is option in the api provider (when loading), can be fixed with descriminatory type , but requires a bit change